articles/concepts/Glossary.md

```diff
@@ -12,31 +12,42 @@ uid: microsoft.quantum.glossary
+|<a id="controlled"></a>Controlled|A quantum operation that takes one or more qubits as enablers for the target operation. For more information, see [Controlled](xref:microsoft.quantum.language.type-model#controlled).|
+|<a id="dirac"></a>Dirac Notation|A shorthand representation of quantum states, for example $\ket{+} = H \ket{0} = \frac{1}{\sqrt{2}} \left(\ket{0} + \ket{1} \right) = \frac{1}{\sqrt{2}}\begin{bmatrix} 1 \\\\  1 \end{bmatrix}$. For more information, see [Dirac Notation](xref:microsoft.quantum.concepts.dirac).|
+|<a id="eigenvalue"></a>Eigenvalue|The factor by which the magnitude of an [eigenvector](xref:microsoft.quantum.glossary#eigenvector) of a given transformation is changed by the application of the transformation. An eigenvalue can be a complex number of any argument. For more information, see [Advanced matrix concepts](xref:microsoft.quantum.concepts.matrix-advanced).|
+|<a id="eigenvector"></a>Eigenvector|A vector whose direction is unchanged by a given transformation and whose magnitude is changed by a factor corresponding to that vector's [eigenvalue](xref:microsoft.quantum.glossary#eigenvalue). An eigenvector can be a complex number of any argument. For more information, see [Advanced matrix concepts](xref:microsoft.quantum.concepts.matrix-advanced).|
+|<a id="entanglement"></a>Entanglement|Quantum particles, such as qubits, can be connected or *entangled* such that actions performed on one immediately duplicate on the other. Entanglement is essential to measuring the state of a qubit.  For more information, see [Advanced matrix concepts](xref:microsoft.quantum.concepts.matrix-advanced).|
+|<a id="epr"></a>EPR pair|One of four specific maximally entangled quantum states of two qubits. The four states are defined $\ket{\beta_{ij}} = (\mathbb{1} \otimes X^iZ^j) (\ket{00} + \ket{11}) / \sqrt{2}$. An EPR pair is also known as a [Bell state](xref:microsoft.quantum.glossary#bell-state)|
+|<a id="evolution"></a>Evolution|How a quantum state changes over time. For more information, see [Matrix exponentials](xref:microsoft.quantum.concepts.matrix-advanced#matrix-exponentials).|
+|<a id="function"></a>Function|A type of subroutine in the Q# language that is purely classical (non-quantum). While functions are used within quantum algorithms, they may not act on qubits or call operations. For more information, see [Operation and function types](xref:microsoft.quantum.language.type-model#operation-and-function-types).|
+|<a id="hadamard"></a>Hadamard|The Hadamard operation (also referred to as the Hadamard gate or transform) acts on a single qubit and puts it in superposition. In Q#, it is the pre-defined [$H$](xref:microsoft.quantum.intrinsic.h) operation.|
+|<a id="gate"></a>Gate|A legacy term for a quantum operation, based on the concept of classical logic gates.|
```

Comment:
Should we say that a gate is a matrix operation (because the gates and operators mentioned in the rest of the glossary are used somewhat interchangeably)?

Reply:
That's part of why I recommend against using "gate" as much as possible. The term is equally often applied to unitary matrices (e.g.: 𝐻) and to intrinsic Q# operations (e.g.: `H`), introducing confusion. Put differently, the definition that you give is correct, @natke, but is one of two mutually contradictory definitions in common use at the moment.